server/views/macros/filterMacro.njk

```diff
@@ -0,0 +1,25 @@
+{% macro categoryFilter(params) %}
```

Comment:
Just realised not in the filter dir - will move in next pr to sit next to the searchFilterMacro